### PR TITLE
[ioctl] move IoAddrToEvmAddr to pkg/util/addrutil/addrutil.go

### LINKED ISSUE #2934: move `IoAddrToEvmAddr` from ioctl/util to pkg/util
**What would you like to be added**:
The code in https://github.com/iotexproject/iotex-core/blob/master/ioctl/util/util.go#L109-L116 could be moved to ./pkg/util/addrutil/addrutil.go, 
And return proper error when conversion doesn't work

**Why is this needed**:

### PR DESCRIPTION
**Pull request by huof6829**
fix #2934

### CODE REVIEW

File: pkg/util/addrutil/addrutil.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package addrutil
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/iotexproject/iotex-address/address"
```
Comment (dustinxie) [inline pkg/util/addrutil/addrutil.go:11]: new line

**Review by dustinxie — APPROVED** [review]

File: pkg/util/addrutil/addrutil.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	"github.com/iotexproject/iotex-core/ioctl/output"
+)
+
+// IoAddrToEvmAddr converts IoTeX address into evm address
+func IoAddrToEvmAddr(ioAddr string) (common.Address, error) {
+	address, err := address.FromString(ioAddr)
+	if err != nil {
+		return common.Address{}, output.NewError(output.ConvertError, "", err)
```
Comment (Liuhaai) [inline pkg/util/addrutil/addrutil.go:19]: This pkg should not depend on the package from ioctl

Comment (dustinxie) [reply pkg/util/addrutil/addrutil.go:19]: +1

**Review by CoderZhi — APPROVED** [review]

File: pkg/util/addrutil/addrutil.go (hunk 2 above)
Comment (CoderZhi) [reply pkg/util/addrutil/addrutil.go:19]: fix this issue before merging

Comment (huof6829) [reply pkg/util/addrutil/addrutil.go]: ok

**Comment by Liuhaai** [comment]
One more substitution (https://github.com/iotexproject/iotex-core/blob/master/api/web3server_utils.go#L89) is needed

**Pull request merged**